orbit/cmd/desktop/desktop.go

```diff
@@ -307,6 +307,9 @@ func main() {
 						IsUnmanaged: sum.Notifications.RenewEnrollmentProfile,
 					})
 
+					// TODO: Use profiles.CheckAssignedEnrollmentProfile here as a
+					// condition of enabling the menu item?
+
```

Comment:
IMO a better place would be https://github.com/fleetdm/fleet/blob/a4cd5e42c48d751fef0d63474423042b785ab997/orbit/pkg/useraction/mdm_migration_darwin.go#L58

the only reason I can think of for not doing the check in Desktop too is the rate limit, but maybe it's not a big deal if the first call busts the cache?

Reply:
I removed this from the scope of this PR since I think we probably want to address it separately.